modal/manifest.json

```
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "9567cd54-7fac-46d8-84ce-0b00cef6937e",
+  "app_id": "modal",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Troubleshooting",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect logs and metrics for your Modal applications",
+    "title": "Modal",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Modal dashboard",
+        "image_url": "images/modal-product-hero.jpg"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Cloud",
+      "Category::Metrics",
+      "Category::Log Collection",
+      "Category::AI/ML",
+      "Submitted Data Type::Metrics",
+      "Submitted Data Type::Logs",
+      "Offering::Integration"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Modal": "assets/dashboards/modal_overview.json"
```

This looks like it's using the wrong path to the dashboard file

```suggestion
      "Modal": "assets/dashboards/modal.json"
```